test/tests/functional/pbs_qsub_direct_write.py

```diff
@@ -47,12 +47,136 @@ class TestQsub_direct_write(TestFunctional):
     def test_direct_write_when_job_succeeds(self):
         """
         submit a sleep job and make sure that the std_files
-        are getting directly written directly to the mapped
-        directory when direct_files option is used.
+        are getting directly written to the mapped directory
+        when direct_files option is used.
+        """
+        j = Job(TEST_USER, attrs={ATTR_k: 'doe'})
+        j.set_sleep_time(10)
```

Comment:
you can reduce the sleep time to a second rather than 10 seconds because copying error/out files is all you are looking to test

Reply:
Hi @arungrover , I am checking file counts for checking for directly written files .
For Example: If job completes within 1 second then both directly written and staged files will be in mapping directory once the job gets completed .

Reply:
In this case please write a comment why longer sleep times are necessary.

Reply:
Instead of a comment, I'd write it in a log message.  It'd help the person running the test to understand why the test is sleeping.